lib/internal/errors.js

```diff
@@ -178,6 +178,16 @@ E('ERR_TRANSFORM_WITH_LENGTH_0',
 E('ERR_UNKNOWN_SIGNAL', 'Unknown signal: %s');
 E('ERR_UNKNOWN_STDIN_TYPE', 'Unknown stdin file type');
 E('ERR_UNKNOWN_STREAM_TYPE', 'Unknown stream file type');
+E('ERR_SOCKET_CLOSED', 'Socket is closed');
+E('ERR_TLS_DH_PARAM_SIZE', (size) =>
+  `DH parameter size ${size} is less than 2048`);
+E('ERR_TLS_HANDSHAKE_TIMEOUT', 'TLS handshake timeout');
+E('ERR_TLS_SESSION_ATTACK', 'TSL session renegotiation attack detected');
+E('ERR_MULTIPLE_CALLBACK',
```

Comment:
This is a redefinition of `ERR_MULTIPLE_CALLBACK` from L162 this causes the error message to change.
1. You can just remove this one and use the one in L162
2. You can remove this one and improve L162 (and update its usages)
3. [Optional] You could sort this whole block alphabetically (very easy if your IDE supports that, otherwise ignore)
4. You can open an issue that `E()` doesn't fail for redefinition, or even better a PR that makes it fail 😉 
```js
not ok 1108 parallel/test-stream-transform-callback-twice
  ---
  duration_ms: 0.61
  severity: fail
  stack: |-
    assert.js:55
      throw new errors.AssertionError({
      ^
    
    AssertionError [ERR_ASSERTION]: 'The undefined callback was called more than once' === 'Callback called multiple times'
        at Transform.<anonymous> (/data/iojs/build/workspace/node-test-commit-linuxone/nodes/rhel72-s390x/test/common/index.js:716:14)
        at emitOne (events.js:115:13)
        at Transform.emit (events.js:210:7)
        at Transform.afterTransform (_stream_transform.js:80:17)
        at Transform.transform [as _transform] (/data/iojs/build/workspace/node-test-commit-linuxone/nodes/rhel72-s390x/test/parallel/test-stream-transform-callback-twice.js:5:37)
        at Transform._read (_stream_transform.js:185:10)
        at Transform._write (_stream_transform.js:173:12)
        at doWrite (_stream_writable.js:371:12)
        at writeOrBuffer (_stream_writable.js:357:5)
        at Transform.Writable.write (_stream_writable.js:274:11)
  ...
```
https://ci.nodejs.org/job/node-test-commit-linuxone/7151/